opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         self.value_type = value_type
         self.enabled = enabled
         self.monotonic = monotonic
+        self.last_update_timestamp = ns_to_iso_str(time_ns())
```

Comment:
I think we should not put that string format in the shared SDK metric type. If the Azure exporter needs that particular format, it should convert the timestamp itself.